src/core/addonManager.js

```diff
@@ -8,14 +8,11 @@ import {
 } from 'core/constants';
 import { addQueryParams } from 'core/utils';
 
-const testHasWindow = () => typeof window !== 'undefined';
-
-export function hasAddonManager({ hasWindow = testHasWindow, navigator } = {}) {
-  // Returns undefined if it cannot be determined if mozAddonManager is supported (likely server
-  // rendering with no window). Otherwise returns true/false based on mozAddonManager in navigator.
-  if (!navigator && !hasWindow()) {
-    return undefined;
+export function hasAddonManager({ navigator } = {}) {
+  if (typeof window === 'undefined') {
```

Comment:
should we still do a check for navigator? is it possible it won't be defined either?

Reply:
It does not matter at this point, because the next condition will be false.

Reply:
🤦‍♂️ yep I see : )